src/etc/init.d/S50dropbear

```diff
@@ -13,25 +13,33 @@ start() {
 
 	echo -n "Starting dropbear sshd: "
 	umask 077
-	# Ensure host keys are changed when instance ID changes
-	cirros-per instance remove-dropbear-host-keys -- rm -rf /etc/dropbear
-	
-	# Make sure dropbear directory exists
-	if [ ! -d /etc/dropbear ]; then
-		mkdir -p /etc/dropbear
+	local dropbear_key_dir="/etc/dropbear"
+
+	# Handle symlinked directories
+	if [ -L "$dropbear_key_dir" ]
```

Comment:
I really much prefer (and is consistent with source code here):

    if [  thing ]; then

to the more vertically-verbose:

    if [ condition ]
    then

Reply:
Oh, right - caught the other style from buildroot's version of this file.